ToolBox/SmartInsider/SmartInsiderConverter.cs

```diff
@@ -240,12 +243,13 @@ private void WriteToFile<T>(DirectoryInfo destinationDirectory, Dictionary<strin
                 var ticker = kvp.Key.ToLowerInvariant();
 
                 var finalFile = new FileInfo(Path.Combine(destinationDirectory.FullName, $"{ticker}.tsv"));
+                var processedFile = new FileInfo(Path.Combine(_processedFilesDirectory.FullName, destinationDirectory.Name, $"{ticker}.tsv"));
                 var fileContents = new List<T>();
 
-                if (finalFile.Exists)
+                if (processedFile.Exists)
                 {
                     Log.Trace($"SmartInsiderConverter.WriteToFile(): Writing to existing file: {finalFile.FullName}");
-                    fileContents = File.ReadAllLines(finalFile.FullName)
+                    fileContents = File.ReadAllLines(processedFile.FullName)
```

Comment:
Not sure I follow how this works, are the logs at 251 and 258 now outdated?
`processedFile` is the combination of `_processedFilesDirectory.FullName` and `destinationDirectory.Name`?

Reply:
`destinationDirectory.Name` gets the `transactions` or `intentions` part of the path, so that we know where to load up the existing files from.

Reply:
thanks for the updated docs - The log at 252 seems to supposed that the `finalFile` is the same as the `processedFile` which isn't the case anymore?

Reply:
I believe it makes sense to keep the logs as-is so that we can differentiate whether we're writing to a new file, or an existing file. Maybe some rewording of that log message would be ideal for further clarity.